src/json.cpp

```diff
@@ -1474,10 +1474,11 @@ void JsonIn::error(std::string message, int offset)
     size_t pos = tell();
     rewind(3, 240);
     size_t startpos = tell();
-    char buffer[241];
-    stream->read(&buffer[0], pos - startpos);
+    char *buffer = new char[pos - startpos + 1];
```

Comment:
Thanks for reminding me the incorrect use of `delete` operation. 

The content of `std::string` is also allocated in heap memory, so there should be no virtual difference if I allocate the char buffer manually.